snowplow-react-native-tracker.podspec

```diff
@@ -0,0 +1,42 @@
+require "json"
+
+package = JSON.parse(File.read(File.join(__dir__, "package.json")))
+folly_compiler_flags = '-DFOLLY_NO_CONFIG -DFOLLY_MOBILE=1 -DFOLLY_USE_LIBCPP=1 -Wno-comma -Wno-shorten-64-to-32'
+
+Pod::Spec.new do |s|
+  s.name         = "snowplow-react-native-tracker"
```

Comment:
the name was `RNSnowplowTracker` before, i don't mind changing it, however i am not sure what are the consequences, if any